src/app/api/bookingDetails/[id]/route.ts

```diff
@@ -0,0 +1,45 @@
+import { Client } from "@notionhq/client";
+
+const notionDatabaseId: string | undefined = process.env.NOTION_DATABASE_ID;
+const notionSecret: string | undefined = process.env.NOTION_SECRET;
+
+// Initialize a new client with the secret
+const notion = new Client({ auth: notionSecret });
+
+export async function GET(request: Request, { params }: { params: { id: string } }) {
```

Comment:
Now you code like `Academy750` creating many files ....

just one file `api/notion.ts` with a singleton, and each of the functions (with a proper name that we can understand it.


getBooking
GetBoat
updateBooking